### DISCUSSION TITLE
Some Tailwind classes missing?

### DISCUSSION THREAD

**Discussion by rikiless** [filamentphp/filament#6226] [General]
Why only some of `mb-*` classes works? For example `mb-2` works, `mb-3 mb-4` not and `mb-6` works. Is Tailwind included in Filament with selected classes only? Is it possible to use all class helpers in Filament?

**Comment by danharrin** [comment]
Like any Tailwind project, we only ship with the classes we use otherwise the css file would be huge. To compile extra classes, set up a theme - https://filamentphp.com/docs/2.x/admin/appearance#building-themes
